README.md

```diff
@@ -1,109 +1,148 @@
+
 ```bash
 bash ~/Downloads/Anaconda3-2019.10-Linux-x86_64.sh
 ```
-A few prompts will pop up. You can type `yes` in all cases. But be careful, one will modify your `bash` scripts. If you
-select `no`, you can still let *Anaconda* change the `shell` variables later with:
+
+A few prompts will appear. You can type `yes` in all cases. But be careful, one of them will modify
+the behavior of your `bash` scripts if you have any. If you select `no`, you can still let
+Anaconda change the `shell` variables later with:
+
 ```
 source <path to conda>/bin/activate
 conda init <type of shell> # bash, zsh, sh, ...
 ```
-In case you affirmatively stepped through each prompt, you can now source your `bashrc` file:
+
+If you accepted all prompts, you can now source your `bashrc` file:
+
 ```bash
 source ~/.bashrc
 ```
-and initialize *Anaconda* with:
+
+and initialize Anaconda with:
+
 ```bash
 conda init
 ```
-For more information, other operating systems or troubleshooting please refer to the [*Anaconda*
-installer site](https://www.anaconda.com/distribution/#linux).
 
-## Create a *conda* environment
-Now comes the provided `environment.yml` file into play. It lists all necessary packages needed
-to use the Jupyter notebooks and the Python files in the Exercises and Solutions folders. Therefore,
-you can simply type (assuming your `shell` has been set up properly by *Anaconda* in the step before):
+For more information, other operating systems or troubleshooting, please refer to the [Anaconda installer site](https://www.anaconda.com/distribution/#linux).
+
+### Create a conda environment
+Now the provided `environment.yml` file comes into play. It lists all necessary packages needed to
+use the Jupyter notebooks and the Python files in the Exercises and Solutions folders. Assuming you
+set up your conda installation properly, you can create a new environment with all required packages
+like this:
+
 ```bash
 conda env create -f environment.yml
 ```
-This will download a bunch of packages and set up the *Anaconda* virtual environment.
-## Activate the environment
-As soon as the virtual environment is created properly (last step) you can start it with:
+
+### Activate the environment
+As soon as the virtual environment is created properly (see last step) you can activate it with:
+
 ```bash
 conda activate introductiontopython
 ```
-## Start the Jupyter notebook server
-Last but not least you will have to start the Jupyter notebook server with:
+
+All the necessary packages should now be available.
+
+### Start the Jupyter notebook server
+Last but not least, you will have to start the Jupyter notebook server with:
+
 ```bash
 jupyter notebook
 ```
 
-# Resources
-The course that resides here in this repository is only the beginning of your journey. There are
-many resources online that are a good reference point to dive deeper into programming with
-Python. We have tried to compile a list of those resouces we find most reasonable.
+## Resources
```

Comment:
the resources section was intended to be a separate chapter on the main level. it includes resources way beyond the scope of the python workshop.

Reply:
Done in latest commit.